packages/typespec-azure-playground-website/samples/azure-core.tsp

```diff
@@ -25,154 +25,154 @@ using Azure.Core.Traits;
   "{endpoint}/widget",
   "Contoso Widget APIs",
   {
-    @doc("""
+    /** 
 Supported Widget Services endpoints (protocol and hostname, for example:
 https://westus.api.widget.contoso.com).
-""")
+ */
     endpoint: string,
   }
 )
 @versioned(Contoso.WidgetManager.Versions)
 namespace Contoso.WidgetManager;
 
-@doc("The Contoso Widget Manager service version.")
+/** The Contoso Widget Manager service version. */
 enum Versions {
-  @doc("Version 2022-08-31")
+  /** Version 2022-08-31 */
   @useDependency(Azure.Core.Versions.v1_0_Preview_2)
   `2022-08-30`,
 }
 
 // Models ////////////////////
 
-@doc("The color of a widget.")
+/** The color of a widget. */
 enum WidgetColor {
-  @doc("Black")
+  /** Black */
```

Comment:
This will not pass lintdiff - you have to add some additional verbage, like 'A Yellow Widget'